polyhost/res/overlay_sources/brand_marks.py

```
@@ -0,0 +1,83 @@
+SI = "https://raw.githubusercontent.com/simple-icons/simple-icons/develop/icons/{}.svg"
+RENDER_PX = 256
+
+
+def _get(url: str, attempts: int = 5) -> bytes:
+    """Fetch with backoff — raw.githubusercontent 403s/resets intermittently
+    under the session network policy, and one failure is not 'unavailable'."""
+    for i in range(attempts):
+        try:
+            return urllib.request.urlopen(
+                urllib.request.Request(url, headers={"User-Agent": "polykybd"}),
+                timeout=30).read()
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)